doc/cephadm/index.rst

```diff
@@ -4,21 +4,37 @@
 Cephadm
 =======
 
-``cephadm`` deploys and manages a Ceph cluster. It does this by connecting the
-manager daemon to hosts via SSH. The manager daemon is able to add, remove, and
-update Ceph containers. ``cephadm`` does not rely on external configuration
-tools such as Ansible, Rook, and Salt.
-
-``cephadm`` manages the full lifecycle of a Ceph cluster. This lifecycle
-starts with the bootstrapping process, when ``cephadm`` creates a tiny
-Ceph cluster on a single node. This cluster consists of one monitor and
-one manager. ``cephadm`` then uses the orchestration interface ("day 2"
-commands) to expand the cluster, adding all hosts and provisioning all
-Ceph daemons and services. Management of this lifecycle can be performed
-either via the Ceph command-line interface (CLI) or via the dashboard (GUI).
-
-``cephadm`` is new in Ceph release v15.2.0 (Octopus) and does not support older
-versions of Ceph.
+``cephadm`` is a utility that is used to manage a Ceph cluster. 
+
+Here is a list of some of the things that ``cephadm`` can do:
+
+- ``cephadm`` can add a Ceph container to the cluster.
+- ``cephadm`` can remove a Ceph container from the cluster.
+- ``cephadm`` can update Ceph containers.
+
+``cephadm`` does not rely on external configuration tools like Ansible, Rook,
+and Salt.  However, those external configuration tools can be used to automate
+operations not undertaken by cephadm itself. To learn more about these external
+configuration tools, visit their GitHub pages:
+
+ * https://github.com/ceph/cephadm-ansible
+ * https://github.com/ceph/ceph-salt
+
```

Comment:
The antecedent list includes Rook, but it isn't in the subsequent bullet list.  Perhaps add
`* https://rook.io`
?